package.json

```diff
@@ -1,31 +1,49 @@
 {
   "name": "@shoutem/theme",
-  "version": "0.11.6",
+  "version": "0.12.0",
```

Comment:
How come you opted for a minor version change? This PR is more-less just code lint reorganization.

Reply:
Because dependencies are being changed and/or added, as well as a feature being added (if we're considering lintability a feature).

`version when you add functionality in a backwards-compatible manner` is the definition of a minor version change.

Reply:
Not sure about lintability being a feature, but I agree that adding dependencies is a valid argument for minor version upgrade